src/content/blog/2025-01-31-ilp-packet-lifecycle.mdx

```diff
@@ -0,0 +1,88 @@
+This year, the humble Interledger Protocol (ILP for short) packet took center stage as Stefan Thomas [gave a keynote](https://www.youtube.com/watch?v=Djw6wMXFv1Q&t=3240s) on how the Interledger Protocol became more fast & reliable by introducing packetization to the protocol.
+With packetization breaking payments up into small amounts, sending a payment no longer had a single failure point and both parties in a payment would be able to manage risk more effectively.
+
+Most integrators of the Interledger protocol are leveraging the ILP packet through [Rafiki](https://github.com/interledger/rafiki), as it provides an efficient way for those integrators to participate in an ILP network.
+Though Rafiki's main purpose is to abstract this, the fresh intrigue around Interledger's transport method invites a look at the connection between Rafiki's high-level payment orchestration to ILP's atomic, low-level concept of the ILP packet.
+
+### Setting the Terms
+Before it starts sending all those ILP packets, Rafiki uses the [Open Payments Standard](https://openpayments.dev/introduction/overview/) to set the parameters of the payment. Interledger doesn't have any mechanism to negotiate payment terms, so Open Payments picks up the slack. 
+
+A prospective debitor will use this standard to set payment terms by first creating an _incoming payment_ on the Rafiki instance hosting the receiving wallet address, then acquiring a _quote_ from the Rafiki instance hosting the sending wallet address.
+Finally, it creates an _outgoing payment_ against the sending wallet address by acquiring consent from its owner to charge them with particular terms such as its amount and its currency.
+Once that is complete, these Rafiki instances can begin to fulfill the terms of this payment using Interledger.
+
+On a more technical point, ILP packets may be involved before an outgoing payment is even created. During the acqusition of a _quote_ from the sending Rafiki instance, it will send a ILP packet accross the network to the destination. This packet will need to traverse each hop on the network as each node's fees and rates won't be known to the sender without doing so.
+
+_For a more on Open Payments, consider reading the [Simple Open Payments Guide](https://interledger.org/developers/blog/simple-open-payments-guide/) on this blog._
+
+### Preparing the Packet
+Now that the sending Rafiki instance has an outgoing payment created against one of its wallet addresses, it can pass the terms of that payment into an Interledger payment. First, it uses the [Simple Payment Setup Protocol](https://github.com/interledger/rfcs/blob/main/0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md) to acquire an ILP address for the receiver, and a shared secret that is used to generate a [_condition_](https://github.com/interledger/rfcs/blob/main/0029-stream/0029-stream.md).
```

Comment:
```suggestion
Now that the sending Rafiki instance has an outgoing payment created against one of its wallet addresses, it can pass the terms of that payment into an Interledger payment. First, it uses the [Simple Payment Setup Protocol](/developers/rfcs/simple-payment-setup-protocol/) (SPSP) to acquire an ILP address for the receiver, and a shared secret that is used to generate a [_condition_](/developers/rfcs/stream-protocol/#6-condition-and-fulfillment-generation).
```